script/build.bat

```diff
@@ -9,6 +9,15 @@ mkdir "%build_dir%"
 echo Webview directory: %src_dir%
 echo Build directory: %build_dir%
 
+:: If you update the nuget package, change its version here
+set nuget_version=1.0.1150.38
```

Comment:
Is this the version of nuget or webview? The variable name is unclear.

Reply:
webview2

Reply:
It's the webview2 nuget package version.